Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365.py

```diff
@@ -497,7 +494,7 @@ def parse_element(element):
     def call(self, email_address, recursive_expansion=False):      # pragma: no cover
         try:
             if recursive_expansion == "True":
-                group_members: Dict = {}
+                group_members: dict = {}
```

Comment:
```suggestion
                group_members = {}
```

Reply:
[2023-07-04 19:03:20,883] [INFO] EWSO365
/builds/xsoar/content/Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365.py:491:17: error:
Need type annotation for "group_members" (hint:
"group_members: Dict[<type>, <type>] = ...")  [var-annotated]
                    group_members = {}
                    ^~~~~~~~~~~~~
Found 1 error in 1 file (checked 1 source file)